---
title: "Analysis of Motif Distributions in Regions of Endocytic Proteins"
author: "Chante Bethell"
subtitle: "Rowan University RUSSS Symposium"
date: "`r format(Sys.time(), '%d %B %Y')`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```

```{r, include=FALSE}
library(tidyverse)
library(scales)
library(cowplot)
library(ggrepel)
library(ggpubr)
data <- read_csv("human_proteome_motifs_across_domains.csv")

filter_count <- function(data, motif) {
  data %>%
  group_by(sequence_id,domain_type,motif_type) %>%
  summarize(count = n()) %>%
  spread(domain_type, count) %>%
  replace_na(list(D = 0, O = 0)) %>%
  arrange(desc(D)) %>%
  filter(motif_type == !!motif)
}

filter_plot <- function(data,motif,number,numx,numy) {
  data %>%
  filter(D > 0) %>%
  ggplot(aes(x = O, y = D)) + 
  scale_x_continuous("Motif Count in Ordered Regions", breaks = pretty_breaks(n = numx), limits=c(0,numx)) +
  scale_y_continuous("Motif Count in Disordered Regions", breaks = pretty_breaks(n = numy), limits=c(0,numy)) +
  labs(title = paste0(motif, " motif counts in protein regions")) +
  geom_hex(bins = number) +
  geom_abline(color="green") + 
  stat_binhex(aes(label=..count..), geom="text", bins=number, colour="white", size=3.5) +
  scale_fill_gradientn(colours=c("red","black"),name = "Frequency",na.value=NA) +
  grids(linetype = "dashed")
}

YXX <- filter_count(data, quo("YXX[LIMFV]"))
```

## Abstract

A short linear motif (SLiM) is a recurring pattern of approximately three to ten amino acids found in proteins. SLiMs are important for cellular signaling and the regulating of proteins, often times by acting as binding sites for protein-binding domains. While SLiMs exist both in ordered regions of proteins where there is a tertiary structure and in disordered regions where there is no structure, they are primarily functional in disordered regions. An important example of SLiM-mediated processes and the focus of this study is endocytosis. Endocytosis is the process by which cells engulf molecules from the extracellular environment. There are specific motifs that mediate and trigger endocytosis. However, the short length of motifs means that it is easy to overlook those that may be important to biological functions. The goal of this study is to identify previously unrecognized proteins that may be involved in endocytosis by analyzing the distribution of motifs in the ordered and disordered regions of the human proteome. Using a bioinformatics approach, we systematically searched the entire human proteome for motifs known to be involved in endocytosis. We hypothesize that the proteins we find to be enriched with motifs in disordered regions may be functionally important for endocytosis. These proteins will be targeted for experimental validation. 

## Introduction

We chose to focus on the motifs within five proteins which are specifically involved in forming clathrin-coated pits for cargo intake in endocytosis: AP2A1, AP2A2, AP2B1, AP2S1, AP2M1, AAK1. According to Koscielny et al., these are, respectively, the 5 subunits of the AP2 protein which triggers clathrin coated pit formation for the binding of cargo, plasma membrane, clathrin, and other accessory proteins (4). While AAK1 is the AP2 associated kinase which phosphorylates this clathrin-mediated endocytosis (1). To be more specific, the activation of AP2 adaptor complex occurs due to the hierarchial binding of the AP2A1, AP2A2, AP2B1, and AP2S1 subunits to phosphatidylinositol-4,5-bisphosphate (PIP2). The stabilization of these active complexes occurs via AP2M1 binding to PIP2 and interactions with YXXPHI-containing cargo (3). The named proteins can be found in Table 1, along with their known function (obtained from the Eukaryotic Linear Motif database) and instances these proteins appear in four motifs that have also been determined to be involved in endocytosis (2). In the Results section, we further show other proteins that we have found to be enriched in motif count in the disordered regions. Each table in this section is headed by the specific motif the table focuses on and has four columns to represent the following:

i. The UNIPROT accession ID for each protein enriched in the number of specified motifs in the disordered region
ii. The ratio of motif counts in the disordered versus ordered regions
iii. Whether or not the protein has been validated and recorded in the Eukaryoitic Linear Motif Database (2). 

The plot following each table represents a heatmap of proteins organized by the distribution of specific motifs in the disordered and ordered regions. The motifs specified were chosen based on their known acitivity in endocytosis, validated by Pandey as follows: X[DE]XXXL[LI], YXX[LIMFV], NPXY, and NPF (5). The AP2 complex is known to recognize the YXX[LIMFV], YXXPhi and X[DE]XXXL[LI] endocytic signal motifs within the cytosolic tails of the transmembrane cargoes (8). In order to study each motif separately, heatmaps and tables were produced for each specific motif. Each hexagon on the heatmaps is labeled with a white number representing the number of proteins that have the same number of motif instances in the ordered and disordered regions, while the red line is a one to one line used to determine which proteins are enriched in the disordered regions and should be targeted based on our hypothesis. Proteins found most above the red line are those that we want to target. 

|UNIPROT Protein Name (ID)| Function | Known Active Motif | Motif Count (Disordered/Ordered) |
| ------------ |:-------------------:| :--------:| :--------------------------:|
|AP2A1 (O95782) |  | X[DE]XXXL[LI] | 0/6 |
|AP2A1 (O95782)|  | YXX[LIMFV] | 0/6 |
|AP2A2 (O94973)|  | X[DE]XXXL[LI] | 0/3 |
|AP2A2 (O94973)|  | YXX[LIMFV] | 0/3 |
|AP2B1 (P63010)| binds to the clathrin heavy chain to promote lattice assembly (in cattle)| X[DE]XXXL[LI] | 0/6 |
|AP2B1 (P63010)| binds to the clathrin heavy chain to promote lattice assembly (in cattle)| YXX[LIMFV] | 0/6 |
|AP2M1 (Q96CW1)|  | X[DE]XXXL[LI] | 0/4 |
|AP2M1 (Q96CW1)|  | YXX[LIMFV] | 0/4 |
|AP2S1 (P53680)|  | YXX[LIMFV] | 0/12 |
|AAK1  (Q2M2I8)|  | NPF | 4/0 |
|AAK1  (Q2M2I8)|  | X[DE]XXXL[LI] | 8/0 |
|AAK1  (Q2M2I8)|  | YXX[LIMFV] | 0/4

```{r, echo=TRUE, warning=FALSE, message=FALSE}
endo_data <- read_csv("endocytosis_involved_proteins.csv")
data <- rename(data, UNIPROT_ID = sequence_id)
data %>%
  filter(UNIPROT_ID == "O95782" | UNIPROT_ID == "O94973" | UNIPROT_ID == "P63010" | UNIPROT_ID == "P53680" | UNIPROT_ID == "Q96CW1" | UNIPROT_ID == "Q2M2I8") %>%
  inner_join(endo_data, by = c('UNIPROT_ID')) %>%
  group_by(UNIPROT_ID,UNIPROT_name,motif_type,domain_type) %>%
  summarize(count = n()) %>%
  spread(domain_type, count) %>%
  replace_na(list(D = 0, O = 0))%>%
  filter(motif_type %in% c("NPF","X[DE]XXXL[LI]","YXX[LIMFV]", "NPXY")) %>%
  ggplot(aes(x = O, y = D)) + 
  scale_x_continuous("Motif Count in Ordered Regions", breaks = pretty_breaks()) +
  scale_y_continuous("Motif Count in Disordered Regions", breaks = pretty_breaks()) +
  geom_point(stat = "identity") +
  geom_abline(color="red") + 
  geom_text_repel(aes(label=UNIPROT_name)) +
  facet_wrap(~motif_type, ncol = 2)
  
```

This plot shows the proteins and their motif count ratios in the specified motifs known to be involved in endocytosis. It shows that the proteins we determined to be active in endocytosis are enriched in motif count in the ordered regions, with the exception of the AAK1 protein kinase which is enriched in motif count in the disordered regions for the NPF and X[DE]XXXL[LI] motifs. 

## Results

### Four motifs known to be active in endocytosis were chosen for closer study as follows:

#### "YXX[LIMFV]" plot
```{r, echo=TRUE, warning=FALSE, message=FALSE}
data <- rename(data, sequence_id = UNIPROT_ID)
filter_plot(YXX, "YXX[LIMFV]", 15, 10, 18)
```

#### "YXX[LIMFV]" table
| UNIPROT accession ID | motif count in disordered regions | motif count in ordered regions | Found in Eukaryotic Linear Motif Database in YXX[LIMFV] motif|
| ------- |:--------:| :--------:| :---------------------------------------:|
| Q9Y4H2  | 17  | 1 | no |
| P35568  | 14  | 0 | no |
| O95487  | 11  | 2 | no |
| Q8WXI7  | 10  | 1 | no |
| B5MCU0  | 9  | 1 | no |
| H7C0E9  | 9  | 1 | no |
| P00533  | 2   | 1 | yes, in Epidermal Growth Factor Receptor |
| P02786  | 0  | 1  | yes, in Transferrin receptor protein 1 which assists the cellular uptake of iron via receptor-mediated endocytosis
| P11279  | 0  | 1  | yes, in Lysosome-associated membrane glycoprotein 1



#### "NPF" plot
```{r, echo=TRUE, warning=FALSE, message=FALSE}
NPF <- filter_count(data, quo("NPF"))
filter_plot(NPF, "NPF", 12, 3, 5)
```

#### "NPF" table
| UNIPROT accession ID | motif count in disordered regions | motif count in ordered regions | Found in Eukaryotic Linear Motif Database in NPF motif|
| ------- |:--------:| :--------:| :---------------------------------------:|
| Q9H1K0  | 6  | 0 | yes, in Rabenosyn-5 which is an effector protein that plays a role in early endocytic membrane fusion |
| Q8NDI1  | 5  | 0 | yes, in EH domain-binding protein 1 which links clathrin-mediated endocytosis to the actin cytoskeleton |
| B5MC86  | 4  | 0 | no |
| P98082  | 4  | 0 | no |
| A0A0D9SGJ6  | 3  | 0 | no |
| O43426  | 4  | 0 | yes, in Synaptojanin-1 which plays a role in clathrin-mediated endocytosis
| Q9Y6I3  | 3  | 0 | yes, in Epsin-1 which assists in the formation of the clathrin-coated pit for endocytosis
| Q04206 | 1  | 0  | yes, in Transcription factor p65
| Q13492 | 1  | 0  | yes, in Phosphatidylinositol-binding clathrin assembly protein
| Q8WXE9 | 2  | 0  | yes, in Stonin-2 which is involved in endocytic machinery, possibly assisting in clathrin-coated vesicle uncoating
| O95208 | 3  | 0  | yes, in Epsin-2 which plays a role in the formation of clathrin-coated pit
| Q9Y6R0 | 1  | 0  | yes, in Numb-like protein
| P49757 | 1  | 0  | yes, in Protein numb homolog
| O95081 | 3  | 0  | yes, in Arf-GAP domain and FG repeat-containing protein 2
| P52594 | 3  | 0  | yes, in Arf-GAP domain and FG repeat-containing protein 1


#### "X[DE]XXXL[LI]" plot
```{r, echo=TRUE, warning=FALSE, message=FALSE}
XD <- filter_count(data, quo("X[DE]XXXL[LI]")) 
filter_plot(XD, "X[DE]XXXL[LI]", 15, 10, 18)
```

#### "X[DE]XXXL[LI]" table
| UNIPROT accession ID | motif count in disordered regions | motif count in ordered regions | Found in Eukaryotic Linear Motif Database in X[DE]XXXL[LI] motif|
| ------- |:--------:| :--------:| :---------------------------------------:|
| Q07283  | 14  | 1 | no |
| Q8N7P7  | 12  | 0 | no |
| A6NEF3  | 11  | 0 | no |
| E7EPM4  | 9  | 0 | no |
| A6NC98  | 5  | 1 | no |
| P09693  | 1  | 1 | yes, in T-cell surface glycoprotein CD3 gamma chain 
| P50895  | 1  | 0 | yes, in Basal cell adhesion molecule|
| P04233  | 1  | 0 | yes, in HLA class II histocompatibility antigen gamma chain |
| P01730  | 0  | 1 | yes, in T-cell surface glycoprotein CD4
| O15118  | 1  | 0 | yes, in NPC intracellular cholesterol transporter 1



#### "NPXY" plot 

```{r, echo=TRUE, warning=FALSE, message=FALSE}
NPXY <- filter_count(data, quo("NPXY"))
filter_plot(NPXY, "NPXY", 15, 10, 18)
```

#### "NPXY" table
| UNIPROT accession ID | motif count in disordered regions | motif count in ordered regions | Found in Eukaryotic Linear Motif Database in NPXY motif|
| ------- |:--------:| :--------:| :---------------------------------------:|
| E9PFD7  | 3  | 0 | no |
| F5GYU8  | 3  | 0 | no |
| K7EJB9  | 3  | 0 | no |
| K7EL50  | 3  | 0 | no |
| P05067  | 1  | 0 | yes, in Amyloid-beta precursor protein |
| P05556  | 2  | 0 | yes, in Integrin beta-1 |
| Q07954  | 1  | 0 | yes, in Prolow-density lipoprotein receptor-related protein 1 which is an endocytic receptor |
| P05106  | 1  | 0 | yes, in Integrin beta-3
| Q9NY15  | 1  | 0 | yes, in Stabilin-1
| Q8TBB1  | 1  | 0 | yes, in E3 ubiquitin-protein ligase LNX
| Q14114  | 1  | 0 | yes, in Low-density lipoprotein receptor-related protein 8 which may function as an endocytic receptor
| P05106  | 1  | 0 | yes, in Integrin beta-3
| P26010  | 1  | 0 | yes, in Integrin beta-7



## Conclusion 

Results do not support the hypothesis. The distribution of motifs in the disordered regions compared to the ordered regions are not always enriched in endocytic proteins as displayed in the first table and plot with the details of the known AP2 complex adaptor proteins and their associated protein kinase, AAK1. In fact, the known endocytic proteins look to be enriched in motif counts in the ordered regions, with exception of AAK1. To our surprise, the functions of the known endocytic proteins were not found in the Eukaryotic Linear Motif database. However, the literature confirms their involvement in clathrin-mediated endocytosis. Furthermore, the results for each endocytic motif differs. For "YXX[LIMFV]", 12969 proteins were found to be enriched in the disordered region and only 1 of these recorded in ELM. For "NPF", 979 proteins were found to be enriched in the disordered region and 12 of these recorded in the ELM (7 of which have a known role in endocytosis). For "X[DE]XXXL[LI]", 7841 protein were found to be enriched in the disordered regions and 5 of these recorded in the ELM. Lastly, for "NPXY", 776 proteins were found to be enriched in the disordered region and 9 of there recorded in the ELM (2 of which have known function in endocytosis). That being said, while the analysis of the specified endocytic proteins may not support the hypothesis, enriched proteins that can be targeted for experimental validation have been identified. 

## References


1. Conner, S. D., & Schmid, S. L. (2002). Identification of an adaptor-associated kinase, AAK1, as a regulator of clathrin-mediated endocytosis. The Journal of Cell Biology, 156(5), 921-929. doi:10.1083/jcb.200108123
2. Diella, F., Via, A., Chica, C., Luck, K., Gould, C., & Gibson, T. (2009). The Eukaryotic Linear Motif Resource (ELM): Regulatory Sites in Proteins. Nature Precedings. doi:10.1038/npre.2009.3152.1
3. Kadlecova, Z., Spielman, S. J., Loerke, D., Mohanakrishnan, A., Reed, D. K., & Schmid, S. L. (2016). Regulation of clathrin-mediated endocytosis by hierarchical allosteric activation of AP2. The Journal of Cell Biology, 216(1), 167-179. doi:10.1083/jcb.201608071
4. Koscielny, A., Malik, A. R., Liszewska, E., Zmorzynska, J., Tempes, A., Tarkowski, B., & Jaworski, J. (2017). Adaptor Complex 2 Controls Dendrite Morphology via mTOR-Dependent Expression of GluA2. Molecular neurobiology, 55(2), 1590â€“1606. doi:10.1007/s12035-017-0436-3
5. Pandey, K. N. (2010). Small peptide recognition sequence for intracellular sorting. Current Opinion in Biotechnology, 21(5), 611-620. doi:10.1016/j.copbio.2010.08.007
6. Tidyverse. (n.d.). Retrieved from https://www.tidyverse.org/
7. Github Motifs Repository.(n.d.). Retrieved from https://github.com/cbethell/motifs
8. (n.d.). Retrieved April 5, 2019, from https://www.genecards.org/cgi-bin/carddisp.pl?gene=AP2M1